db_schema/schema.sql

```diff
@@ -530,6 +530,15 @@ INSERT INTO appleCountries (id, name, code, ccc) VALUES (4, "Afghanistan", "AF",
 (894, "Zambia", "ZM", "zmb"),
 (716, "Zimbabwe", "ZW", "zwe");
 
+CREATE TABLE IF NOT EXISTS anchorEpisodesPage (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  web_episode_id VARCHAR(128) NOT NULL,
+  PRIMARY KEY (account_id, episode_id, web_episode_id)
+);
+
+CREATE INDEX idx_account_episode ON anchorEpisodesPage(account_id, episode_id, web_episode_id);
```

Comment:
the PK is also an index, so this is not needed. a different order makes sende to resolve web_ep_ids to episode_ids `INDEX (account_id, web_episode_id, episode_id)`

Reply:
Ah, got it. I've added a comment as well, because I did not know about that MySQL feature.